### PR TITLE
refactor: Update libraries, and jitpack config

### PR DESCRIPTION
**Pull request by 8cAyqpVKio**
The build has already been verified on jitpack.io, and tested on the Tachiyomi app.
[](https://jitpack.io/#8cAyqpVKio/image-decoder)
[](https://github.com/8cAyqpVKio/tachiyomi/actions/runs/3455366088)

Please review.

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 android {
-    compileSdkVersion 30
-    buildToolsVersion "30.0.3"
+    compileSdkVersion 33
+    ndkVersion "25.1.8937393"
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 30
+        minSdkVersion 23
```
Comment (arkon) [inline library/build.gradle]: Why the increase?

File: library/build.gradle [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+        targetSdkVersion 29
```
Comment (arkon) [inline library/build.gradle]: Why the decrease?

File: library/build.gradle (hunk 1 above)
Comment (8cAyqpVKio) [reply library/build.gradle]: matched with tachiyomi-app 's version.
no other reason

Comment (8cAyqpVKio) [reply library/build.gradle]: fixed

File: library/build.gradle (hunk 2 above)
Comment (8cAyqpVKio) [reply library/build.gradle]: fixed

**Comment by 8cAyqpVKio** [comment]
jitpack.io CI is down. Cannot build now :(

edit: build finished
[](https://jitpack.io/#8cAyqpVKio/image-decoder)
We can test on tachiyomi-app like this.
https://github.com/tachiyomiorg/tachiyomi/compare/master...8cAyqpVKio:tachiyomi:master#diff-697f70cdd88ba88fe77eebda60c7e143f6ad1286bca75017421e93ad84fb87df

**Comment by arkon** [comment]
Seems fine from my limited testing, so I'll merge it. We'll see if there's any reports in preview builds with the updates.

**Pull request merged**